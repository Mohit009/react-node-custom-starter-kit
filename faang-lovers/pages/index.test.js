import React from 'react';
import renderer from 'react-test-renderer';
import ComponentUnderTest from './index';

test('Link changes the class when hovered', () => {
  const component = renderer.create(
    <ComponentUnderTest />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});