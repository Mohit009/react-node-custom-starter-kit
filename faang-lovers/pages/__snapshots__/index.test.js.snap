// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<div>
  <main
    className="makeStyles-main-1"
  >
    Welcome to Kartik-Mohit
    <div>
      <h1>
        Anywhere in your app!
      </h1>
      <form
        onSubmit={[Function]}
      >
        <div>
          <label>
            Email: 
          </label>
          <input
            name="email"
            onBlur={[Function]}
            onChange={[Function]}
            type="email"
            value=""
          />
          <span
            style={
              Object {
                "color": "red",
              }
            }
          />
        </div>
        <div>
          <label>
            Pass: 
          </label>
          <input
            name="password"
            onBlur={[Function]}
            onChange={[Function]}
            type="password"
            value=""
          />
        </div>
        <button
          disabled={false}
          type="submit"
        >
          Submit
        </button>
      </form>
    </div>
  </main>
  <footer />
</div>
`;
